### DISCUSSION TITLE
Motu

### DISCUSSION THREAD

**Discussion by matteopomiato** [elrumo/macOS_Big_Sur_icons_replacements#1363] [Icon request]
Can i ask you if is possible to create Motu DP (10,11) icon? Many thanks, your job is great!

**Comment by di0xazin-e** [comment]
Hello!

I didn't have any other icons that I had ideas for icons so I decided to go on this offer! I did a very simple design but if you want better then let me know! (From the time I text this, I hope they are approved soon).

https://macosicons.com/#/u/alivat0r

**Comment by matteopomiato** [comment]
They are so cool! Many thanks man! ♥️🙏🏻

Il giorno dom 6 mar 2022 alle 22:33 a Tečh høE ***@***.***>
ha scritto:

**Reply by di0xazin-e** [reply]
Of course! Always happy to help!
